packages/react/tooltip/src/Tooltip.tsx

```diff
@@ -33,10 +33,11 @@ const stateMachine = createStateMachine(tooltipStateChart);
 const TOOLTIP_NAME = 'Tooltip';
 
 type TooltipContextValue = {
-  triggerRef: React.RefObject<HTMLButtonElement>;
   contentId: string;
   open: boolean;
   stateAttribute: StateAttribute;
+  trigger: HTMLButtonElement | null;
+  onTriggerChange(trigger: HTMLButtonElement | null): void;
```

Comment:
Probably doesn't need to be specifically a button element type?

Reply:
I just copied what was there before

```diff
- triggerRef: React.RefObject<HTMLButtonElement>;
+ trigger: HTMLButtonElement | null;
```

Would you prefer I change it to `HTMLElement` ?

Reply:
In reality I don't think it really matters cause it's just internal…
Because I don't know if HTMLElement is better really as we know it should at least be an interactive element…

Reply:
Oh, shall I change it back to `HTMLButtonElement` then?

Reply:
Yeah I guess it probably carries the intent better considering that it matches the default tag for that part…

Reply:
Sorry about that 😀

Reply:
Maybe I should change it to `React.ElementRef<typeof TooltipTrigger>` like we do elsewhere?

Reply:
Oh I like that yeah. I don't think we've done this consistently but yeah definitely the way to go.